### PR TITLE
fix #535 observer in hosted games Tooltip

### LINKED ISSUE #535: Player VS VS
**I experienced...** hovering over a game in the games tab with 3 players showed this:
untitled

**how to reproduce** not sure

**I expected...** not a second VS and actually a list of the other players in the game.

**Where the bug is / What should be done** if you have a suggestion for fixing the issue, add it here.

### PR DESCRIPTION
**Pull request by GrotheFAF**
collecting observers in a separate list not as one of the teams

- [ ] PR branch should be named `issuenum`-`fix`/`feature`/`cleanup`-`description`
- [ ] Code is split into logical commits
- [ ] Code has tests
- [ ] Final commit includes "Fixes #issue" in commit message

When all builds pass and a maintainer is happy with your PR, the "ready" label will be applied. Please complete these tasks then:

- [ ] Rebase onto develop
- [ ] Add changelog entry
- [ ] Remove this entire template section

### CODE REVIEW

**Comment by GrotheFAF** [comment]
I don't know #709 (and I don't know what backported means)

**Comment by Wesmania** [comment]
It's similar to changes I made in 537eccde90a3e5f57158dcc2161f86886f72ba07 in #709, I assumed you lifted it from there. Nevermind.

**Comment by GrotheFAF** [comment]
How did you get to such an assuption? And why would I do that? You're pretty twisted.
Remember #591 ?

File: src/games/gameitem.py [hunk 1]
```diff
@@ -298,58 +297,50 @@ def update(self, message, old_client=None):
...
 
-    def editTooltip(self, teams):
-        
-        observerlist = []
+    def editTooltip(self, teams, observerlist):
+
         teamlist     = []
 
         teams_string = ""
+        for i, team in enumerate(teams):
```
Comment (Wesmania) [inline src/games/gameitem.py]: Adding to that previous comment - I checked out of interest, you actually can enumerate from something else than 0 if you give it the starting number as the second argument. Didn't even know that.

**Comment by Wesmania** [comment]
It does seem that the team names sent from the server are strings instead of ints, and u'null'. That's so weird.

**Comment by Wesmania** [comment]
+1, just edit the commit message to be more informative

**Comment by GrotheFAF** [comment]
done 
(didn't realized that that would overwrite the old one message)

**Comment by GrotheFAF** [comment]
add changelog

**Comment by GrotheFAF** [comment]
it is merged.
(a commit during merge, prevented to show it as merged)

**Pull request closed without merging**